versions/1.0/index.md

```diff
@@ -354,6 +354,36 @@ A `Collection` **MAY** have the following:
 }
 ```
 
+## <a href="#document-components-collection" id="document-components-collection" class="headerlink"></a> Partial Collection (Optional)
+Server **MAY** support an ability to return only specific fields requested by client.   
+A `PartialCollection` is a type of [node](#document-components-node) used to represent a specific resource fields based on query string parameter
```

Comment:
As per our discussion, i think this should become its own section called `Sparse Fields` or something along those lines, instead of creating a PartialCollection. This will allow clients to filter resources by attributes while still maintaining _what_ resource they are filtering.

ex.

```json
{
    "@id": "/users?fields=/given_name",
    "@type": "Collection",
    "items": [
        {
            "@id": "/users/1",
            "@type": "User",
            "given_name": "Hubert"
        },
        {
            "@id": "/users/2",
            "@type": "User",
            "given_name": "John"
        }
        ...
    ],
    "total_items": 20
}

Reply:
The way jsonapi does this is similar: https://jsonapi.org/format/#fetching-includes, though a difference here is that `includes` is _additional data_ that would be fetched if the param was present. Just another example to look at.

Reply:
Looks like they've added sparse fieldsets to jsonapi now too: https://jsonapi.org/format/#fetching-sparse-fieldsets